### DISCUSSION TITLE
Having trouble with rnote on weblate

### DISCUSSION THREAD

**Discussion by astro-ray** [flxzt/rnote#757] [Q&A] [answered]
I use weblate to translaete rnote, but today I noticed all the data of rnote in weblate is gone. Can someone help?

**Comment by flxzt** [comment, marked as answer]
Thanks for letting me know. The weblate repository needed to be reset after moving the translation files into a different directory in the source repo (#755). It should work again now.

**Reply by astro-ray** [reply]
Maybe you should update the [README.md](https://github.com/flxzt/rnote#translations) file too
Now it should be `./crates/rnote-ui/po`

![image](https://github.com/flxzt/rnote/assets/69363775/4e484a39-e50e-4256-ab26-43e390eb3f29)

**Reply by flxzt** [reply]
thats true, done!
